pkg/registrationserver/health_check_handler.go

```diff
@@ -0,0 +1,13 @@
+package registrationserver
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// HealthCheckHandler returns a default heath check result.
```

Comment:
This needs more detailing and a mock mode. But I can adapt my PR for this after this one is merged.